Header header
uint8 shuttleType
int32 product
int32 firstDestination
